e2e/specs/st_markdown.spec.js

```diff
@@ -28,7 +28,7 @@ describe("st.markdown", () => {
       expect(els[2].textContent).to.eq("This HTML tag is not escaped!");
       expect(els[3].textContent).to.eq("[text]");
       expect(els[4].textContent).to.eq("link");
-      expect(els[5].textContent).to.eq("");
+      expect(els[5].textContent).to.eq("[][]");
```

Comment:
This unit test has been broken. The old version of `react-markdown` had a bug that incorrectly rendered this case.

Reply:
Huh, interesting. I took a look at the [Git blame](https://github.com/akrolsmir/streamlit/commit/1c35e834c72a2482abab84f9742df126d455bedf) where we introduced this unit test, but intuitively it makes sense that "[][]" should render as that and not empty string, in Markdown.